.goreleaser.yml

```
@@ -17,20 +17,26 @@ builds:
       - CC=o64-clang
       - CXX=o64-clang++
     ldflags:
-      -s -w -X 'github.com/${GITHUB_REPOSITORY}/versioning.Version=v{{ .Version }}'
+      -s -w -X 'github.com/0xPolygon/polygon-edge/versioning.Version=v{{ .Version }}'
+      -s -w -X 'github.com/0xPolygon/polygon-edge/versioning.Commit={{ .Commit }}'
+      -s -w -X 'github.com/0xPolygon/polygon-edge/versioning.Branch={{ .Branch }}'
+      -s -w -X 'github.com/0xPolygon/polygon-edge/versioning.BuildTime={{ .Date }}'
```

No need for providing -s and -w multiple times. These flags are for removing symbols and debug information from the binary:

```suggestion
      -s -w 
      -X 'github.com/0xPolygon/polygon-edge/versioning.Version=v{{ .Version }}'
      -X 'github.com/0xPolygon/polygon-edge/versioning.Commit={{ .Commit }}'
      -X 'github.com/0xPolygon/polygon-edge/versioning.Branch={{ .Branch }}'
      -X 'github.com/0xPolygon/polygon-edge/versioning.BuildTime={{ .Date }}'
```